### DISCUSSION TITLE
Problem: the remix project fails to include the justd cli.

### DISCUSSION THREAD

**Discussion by dabhivijay2478** [irsyadadl/intentui#245] [General]
the remix project fails to include the justd cli. 

![image](https://github.com/user-attachments/assets/92ff344c-9451-4d62-b1c3-7a9544a65e65)

**Comment by irsyadadl** [comment]
Can you please make a workflow. Like how you setup your remix and follow running justd init.

**Reply by dabhivijay2478** [reply]
I created a project using npx create-remix. After that, I installed the JustD CLI, but I couldn't install it properly.

**Reply by irsyadadl** [reply]
Thanks, I'll update later, but for now. Just remove the comment
```diff
-// Vite takes care of building everything, not tsc.
```
